Tests/CombineTests/PublisherExtensionsTests.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  PublisherExtensionsTests.swift
+//  SwifterSwift
+//
+//  Created by Heecheon Park on 7/28/23.
+//  Copyright © 2023 SwifterSwift
+//
+
+// swiftlint:disable identifier_name large_tuple
+
+@testable import SwifterSwift
+import XCTest
+
+#if canImport(Combine)
+import Combine
+
+final class PublisherExtensionsTests: XCTestCase {
+
+    func testCombineLatest() throws {
+        if #available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *) {
```

Comment:
Use `@available` for the class, same as you do in the implementation.

Reply:
handled at c66c7fb4bb52f15ec1c5b34be6c90ef77e380ad5

Reply:
You can remove the `if #available` now.